rectified: true
rectifiedB: true

numOfFrames: 978
seq: "00"

multi: true

gtPose : true

leftPathCam : "/camera_1/left/image_rect"
rightPathCam : "/camera_1/right/image_rect"

leftPathCamB : "/camera_4/left/image_rect"
rightPathCamB : "/camera_4/right/image_rect"

gtPath : "ground_truth/pose"

imagesPath: "/home/christos/catkin_ws/src/mini_project_kokas/src/vio_slam/images/simulation/"

fileExtension: ".jpg"

Camera_l:
  fx: 263.2786443617774
  fy: 263.2786443617774
  cx: 376.5
  cy: 240.5
  k1: 0
  k2: 0
  p1: 0
  p2: 0
  k3: 0

Camera_r:
  fx: 263.2786443617774
  fy: 263.2786443617774
  cx: 376.5
  cy: 240.5
  k1: 0
  k2: 0
  p1: 0
  p2: 0
  k3: 0

Camera:
  width: 752
  height: 480
  fps: 15.0
  bl: 0.12

CameraB_l:
  fx: 263.2786443617774
  fy: 263.2786443617774
  cx: 376.5
  cy: 240.5
  k1: 0
  k2: 0
  p1: 0
  p2: 0
  k3: 0

CameraB_r:
  fx: 263.2786443617774
  fy: 263.2786443617774
  cx: 376.5
  cy: 240.5
  k1: 0
  k2: 0
  p1: 0
  p2: 0
  k3: 0

CameraB:
  width: 752
  height: 480
  fps: 15.0
  bl: 0.12

Camera_path: /zed2i/zed_node/left_raw/image_raw_color
ground_truth_path: /odom/ground_truth/2
pointcloud_path: /camera_1/points2

IMU:
  T_b_c1: 
    rows: 4
    cols: 4
    dt: f
    data: [  0.0040000, -0.0000080,  0.9999920, -0.008,
            -0.9999840,  0.0040000,  0.0040000, 0.023,
            -0.0040000, -0.9999920,  0.0000080, -0.000,
            0.0, 0.0, 0.0, 1.0]
  path: /imu/data

Stereo:
  T_c1_c2:
    rows: 4
    cols: 4
    dt: f
    data: [1.0, 0.0, 0.0, -0.12,
         0.0, 1.0, 0.0, 0.0,
         0.0, 0.0, 1.0, 0.0,
         0.0, 0.0, 0.0, 1.0]

Multi:
  T_c1_c2:
    rows: 4
    cols: 4
    dt: d
    data: [-1.0, 0.0, 0.0, 0.12,
         0.0, 1.0, 0.0, 0.0,
         0.0, 0.0, -1.0, -0.3,
         0.0, 0.0, 0.0, 1.0]

T_w_c1:
  rows: 4
  cols: 4
  dt: d
  data: [0.0, 0.0, 1.0, 0.06,
        -1.0, 0.0, 0.0, 0.0,
        0.0, -1.0, 0.0, 0.2,
        0.0, 0.0, 0.0, 1.0]

# T_w_c1:
#   rows: 4
#   cols: 4
#   dt: d
#   data: [0.0, 0.0, 1.0, 0.150,
#         -1.0, 0.0, 0.0, 0.06,
#         0.0, -1.0, 0.0, 0.25,
#         0.0, 0.0, 0.0, 1.0]